src/Microsoft.TestPlatform.Common/ProcessCallbacks.cs

```diff
@@ -1,17 +1,18 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
-namespace Microsoft.TestPlatform.TestHostProvider.Hosting
+namespace Microsoft.VisualStudio.TestPlatform.Common
```

Comment:
HostProviderEventArgs is interesting for DataCollectors? Can that not stay along side TestHostProvider implementation?